src/Version.cpp

```diff
@@ -34,28 +35,36 @@
 namespace espressopp {
 
   Version::Version() {
-	  name          = "ESPResSo++";
-      major         = MAJORVERSION;
-	  minor         = MINORVERSION;
-	  patchlevel    = PATCHLEVEL;
-	  gitrevision    = gitversion;
-      boostversion  = BOOST_LIB_VERSION;
-	  date          = __DATE__;
-	  time          = __TIME__;
+    name_          = "ESPResSo++";
+    major_         = MAJORVERSION;
+    minor_         = MINORVERSION;
+    patchlevel_    = PATCHLEVEL;
+    gitrevision_    = gitversion;
+    boostversion_  = BOOST_LIB_VERSION;
+    date_          = __DATE__;
+    time_          = __TIME__;
```

Comment:
1. clean up, there was a mix of tabs and spaces.
2. the variable name 'name_' allows me to have ver.name() in the code, of course that's a matter of taste, nothing ore. (following: https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Variable_Names)